app/templates/index.hbs

```diff
@@ -0,0 +1,9 @@
+<div>hello</div>
+{{#each model as |reminder|}}
```

Comment:
This will display all of the reminders which should be in the reminders template. Also a <ul> tag should be around the each allowing all the <li>'s to be in the <ul>